pkgs/development/tools/misc/icon-naming-utils/default.nix

```diff
@@ -8,14 +8,7 @@ stdenv.mkDerivation rec {
     sha256 = "071fj2jm5kydlz02ic5sylhmw6h2p3cgrm3gwdfabinqkqcv4jh4";
   };
 
-  buildInputs = [ librsvg ] ++ (with perlPackages; [ perl XMLSimple ]);
-
-  postInstall =
```

Comment:
I removed the `substituteInPlace` because the shebang is already fixed in the fixup phase.
I assume the hook is picking up the (now right) perl interpreter from the buildInputs.